content/blog/posts/2022-05-19-five-tips-to-improve-your-terminal-experience.md

```diff
@@ -0,0 +1,70 @@
+---
+authorId: rwwagner90
+categories:
+  - devtools
+date: '2022-05-19'
+slug: five-tips-to-improve-your-terminal-experience
+title: 5 Tips to Improve Your Terminal Experience
+---
+
+We've come a long way from the stereotypical terminals seen in the movies with
+green text on black backgrounds and developers furiously
+slinging commands from memory - all on an eight inch screen.
+
+There are now lots of great options for improving both the style and functionality
+of your terminal, to provide you with the best development experience possible.
+
+# Fish
+
+[![ASCII art depicting a fish](/img/blog/five-tips-to-improve-your-terminal-experience/fish.png)](https://fishshell.com/)
+
+One of the biggest improvements to my experience in the terminal was from switching from `bash` to
+[fish](https://fishshell.com/). Fish upgrades your terminal with features like autosuggestions, that
+remember commands you have run before and try to auto complete them for you, and automatically
```

Comment:
syntax suggestions

"...which remembers commands you have run before and tries to autocomplete them for you. This feature also generates new autocompletes for other utilities by parsing their man pages. Neat!"